pkg/server/apidev/apidev.go

```diff
@@ -472,12 +477,12 @@ func LogsHandler(ctx context.Context, state *state.State, r *http.Request, flush
 	vars := mux.Vars(r)
 	runID, ok := vars["run_id"]
 	if !ok {
-		return errors.Errorf("Run id was not supplied, request path must be of the form /dev/logs/<run_id>")
+		return libhttp.NewErrBadRequest("run id was not supplied")
 	}
 
 	run, ok := state.Runs.Get(runID)
 	if !ok {
-		return errors.Errorf("Run with id %s not found", runID)
+		return libhttp.NewErrNotFound("Run with id %s not found", runID)
```

Comment:
nit: lowercase error msgs for wrapping

```suggestion
		return libhttp.NewErrNotFound("run with id %q not found", runID)
```